media/js/firefox/firstrun/firstrun_quantum.js

```diff
@@ -5,6 +5,11 @@
 (function (Mozilla) {
     'use strict';
 
+    // Add class to indicate Windows 8 and later
+    if ((window.site.getPlatform() === 'windows') && (parseFloat(window.site.getPlatformVersion()) >= 6.2)) {
```

Comment:
You can simplify this to `if (window.site.platform === 'windows' && window.site.platformVersion >= 6.2)`